### PR TITLE
feat: Data Owner Implementation of Atlas Proxy

### PR DESCRIPTION
**Pull request by verdan**
### Summary of Changes

_Include a summary of changes then remove this line_

### Tests

_What tests did you add or modify and why? If no tests were added or modified, explain why. Remove this line_

### Documentation

_What documentation did you add or modify and why? Add any relevant links then remove this line_

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [ ] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [ ] PR includes a summary of changes. 
- [ ] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does
- [ ] PR passes `make test`

### CODE REVIEW

File: metadata_service/proxy/atlas_proxy.py [hunk 1]
```diff
@@ -35,6 +35,11 @@
 _ATLAS_PROXY_CACHE_EXPIRY_SEC = 11 * 60 * 60 + randint(0, 3600)
 
 
+class STATUS:
```
Comment (mgorsk1) [inline metadata_service/proxy/atlas_proxy.py:38]: please use camelcase and maybe subclass Enum ?

Comment (verdan) [reply metadata_service/proxy/atlas_proxy.py:38]: Oh nice catch, it should be Title Case.

File: metadata_service/proxy/atlas_proxy.py [hunk 2]
```diff
@@ -418,7 +439,7 @@ def get_table(self, *, table_uri: str) -> Table:
                 name=attrs.get('name') or table_qn.get("table_name", ''),
                 tags=tags,
                 description=attrs.get('description') or attrs.get('comment'),
-                owners=[User(email=attrs.get('owner'))],
+                owners=self._get_owners(table_details[self.REL_ATTRS_KEY].get("ownedBy"), attrs.get('owner')),
```
[the remark this answers is not in the document]
Comment (verdan) [reply metadata_service/proxy/atlas_proxy.py:442]: okay 😢

**Comment by verdan** [comment]
@feng-tao @jinhyukchang @mgorsk1 can you please review this PR?

**Comment by feng-tao** [comment]
will leave it to @mgorsk1 :)

File: metadata_service/proxy/atlas_proxy.py [hunk 3]
```diff
@@ -372,6 +377,24 @@ def _get_reports(self, guids: List[str]) -> List[ResourceReport]:
 
         return parsed_reports
 
+    def _get_owners(self, data_owner: list, fallback_owner: str) -> List[User]:
```
Comment (mgorsk1) [inline metadata_service/proxy/atlas_proxy.py:380]: if data_owner is a list maybe it should be data_owners ?

File: metadata_service/proxy/atlas_proxy.py [hunk 4]
```diff
@@ -372,6 +377,24 @@ def _get_reports(self, guids: List[str]) -> List[ResourceReport]:
...
+        data_owners = list()
```
Comment (mgorsk1) [inline metadata_service/proxy/atlas_proxy.py:381]: could be renamed to `result` (based on change data_owner > data_owners in method args)

File: metadata_service/proxy/atlas_proxy.py (hunk 3 above)
Comment (verdan) [reply metadata_service/proxy/atlas_proxy.py]: DONE

File: metadata_service/proxy/atlas_proxy.py (hunk 4 above)
Comment (verdan) [reply metadata_service/proxy/atlas_proxy.py]: changed to `owners_detail`

**Comment by verdan** [comment]
@feng-tao good to merge this ?

**Pull request merged**